docs/troubleshooting/running-tests.md

```diff
@@ -18,125 +12,104 @@ If you’re missing a simulator, make sure Xcode is installed and use it to down
 
```

Comment:
The screenshot above (with how to install iOS 9.1 runtime) is very old, needs to be updated.

Reply:
Great catch. Rephrased and changed link to apple guide - https://developer.apple.com/documentation/safari-developer-tools/adding-additional-simulators